product-service/src/db/products.js

```diff
@@ -0,0 +1,103 @@
+import createError from 'http-errors';
+import { createClient } from './config';
+import { errorResponse } from '../utils/api';
+
+export const getProductFromDB = async () => {
+  const dbClient = createClient();
```

Comment:
Would be great to do not create db client each time you are interacting with db.

You might create DB connection in lambda memory to be reused in between multiple calls using keep alive functionality as a probable optimization with single db client approach